packages/store-ui/src/molecules/IconButton/stories/IconButton.mdx

```diff
@@ -0,0 +1,18 @@
+import { Story, Canvas, ArgsTable } from '@storybook/addon-docs'
+import IconButton from '../IconButton'
+
+# Preview
```

Comment:
I believe in another PR it was decided that "default" should be used
```suggestion
# Default
```

Reply:
Yeah, i will change

Reply:
Fixed